tools/rsictl/src/subcmds.rs

```diff
@@ -185,3 +189,10 @@ pub(crate) fn sealing_key(args: &SealingKey) -> GenericResult
 
     Ok(())
 }
+
+pub(crate) fn realm_metadata() -> GenericResult
+{
+    let realm_metadata = rust_rsi::realm_metadata()?;
+    hexdump(&realm_metadata[0..SIGNED_METADATA_SIZE] , 16, Some("Realm metadata"));
```

Comment:
I think it would be nice to do some pretty printing of the metada struct instead of just making a hexdump. Rsictl when asked for attestation token prints out every field in a nice, human readable form.